apps/protocol-frontend/src/components/ContributionTypesTable.tsx

```diff
@@ -59,7 +59,7 @@ const ContributionTypesTable = ({
         date_of_submission: contributionType.date_of_submission,
         date_of_engagement: contributionType.date_of_engagement,
       })),
-    [],
+    [], // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
May I ask why disabling? 
Shouldn't this react to unique contributions changes?

Reply:
An empty array means the useEffect is called once and it is a common practice. We won't run into the same issues as a useEffect with partial dependencies